final/problem-def-ana/problem-def.tex

```diff
@@ -1,10 +1,8 @@
 \section{Problem Definition}
 \label{sec:problem-definition}
 
-The previous sections provided the necessary background knowledge of
-the problem domain. Now that the background knowledge has been
-explained, this section will discuss the problem definition of the
-client.
+Now that the background knowledge has been explained, this section discusses the
+problem definition of the client.
```

Comment:
I think you can just remove the first part of the sentence: "Now that the necessary background knowledge has been explained..."

So maybe something like this:

> In this section, the problem of the client is defined. First the general problem is given in the form of a question. After that the general definition will be divided into multiple sub-definitions.